### PR TITLE
duck 7.6.0

### PR DESCRIPTION
**Pull request by dkocher**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

```
  * C: 1: col 1: Missing top-level class documentation comment.
  * C: 1: col 1: Missing frozen string literal comment.
  * C: 35: col 19: Align the arguments of a method call if they span more than one line.
Error: 3 problems in 1 formula detected
```

### CODE REVIEW

**Comment by bayandin** [comment]
```
==> FAILED
Error: 1 problem in 1 formula detected
duck:
  * 'revision 1' should be removed
```

File: Formula/duck.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Duck < Formula
   desc "Command-line interface for Cyberduck (a multi-protocol file transfer tool)"
   homepage "https://duck.sh/"
   # check the changelog for the latest stable version: https://cyberduck.io/changelog/
-  url "https://dist.duck.sh/duck-src-7.5.2.33336.tar.gz"
-  sha256 "f084aca8b58069ccfeb62752f47daeff80f54991478189b274c06c7cfe0fa338"
+  url "https://dist.duck.sh/duck-src-7.6.0.33437.tar.gz"
+  sha256 "b724ee37fce1cf1d49c07f0d765ff2d233cb58a6b09c6d2c6ae0870061c157d2"
   license "GPL-3.0-only"
   revision 1
```
Comment (bayandin) [inline Formula/duck.rb:8]:
```suggestion
```

```
==> FAILED
Error: 1 problem in 1 formula detected
duck:
  * 'revision 1' should be removed
```

**Review by SMillerDev — DISMISSED** [review]

**Comment by dkocher** [comment]
Not sure why tap_syntax failed. Can this be re-run?

**Comment by bayandin** [comment]
> Not sure why tap_syntax failed. Can this be re-run?

Now it's green 🎉 

Could you please squash all the commits into a single one with a message `duck 7.6.0`

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**